project-base/assets/js/frontend/components/Login.js

```diff
@@ -38,13 +38,13 @@ export default class Login {
                 } else {
                     const $validationErrors = $('.js-window-validation-errors');
                     if ($validationErrors.hasClass('display-none')) {
-                        if (data.message === undefined) {
+                        if (data.reason === 'too-many-attempts') {
```

Comment:
I looks like upgrade notes are needed...?

Reply:
merged with the previous upgrade note entry